lms/static/scripts/frontend_apps/components/test/FilePickerFormFields-test.js

```diff
@@ -44,35 +40,13 @@ describe('FilePickerFormFields', () => {
   });
 
   describe('content_items field', () => {
-    it('renders content_items field for URL content', () => {
+    it('renders content_items field for content', () => {
+      const content = { type: 'url', url: 'https://example.com/' };
       const formFields = createComponent({
-        content: { type: 'url', url: 'https://example.com/' },
+        content,
       });
       const contentItems = getContentItem(formFields);
-      assert.deepEqual(
-        contentItems,
-        contentItemForUrl(launchURL, 'https://example.com/')
-      );
-    });
-
-    it('renders content_items field for LMS file content', () => {
-      const file = { id: 123 };
-      const formFields = createComponent({
-        content: { type: 'file', file },
-      });
-      const contentItems = getContentItem(formFields);
-      assert.deepEqual(contentItems, contentItemForLmsFile(launchURL, file));
-    });
-
-    it('renders content_items field for VitalSource book content', () => {
-      const formFields = createComponent({
-        content: { type: 'vitalsource', bookID: 'test-book', cfi: 'test-cfi' },
-      });
-      const contentItems = getContentItem(formFields);
-      assert.deepEqual(
-        contentItems,
-        contentItemForVitalSourceBook(launchURL, 'test-book', 'test-cfi')
-      );
+      assert.deepEqual(contentItems, contentItemForContent(launchURL, content));
```

Comment:
Instead of this assertion, I suggest to check that the hidden form has the expected value property.

Reply:
I'm not quite sure I've understood. Checking the `value` property of the `content_item` field is what this test is doing, albeit with some indirection. Maybe you were expecting something like:

```js
const value = wrapper.find('[name="content_item"]'.prop('value');
assert.equal(value, "some string");
```

I'm using `contentItemForContent(...)` to generate the expected value since the structure is complex enough to make it worthwhile. I've inlined the `getContentItem(...)` helper though to make the test a bit more direct.

Reply:
Indeed.